src/visitable.cpp

```diff
@@ -12,6 +12,10 @@
 #include "string_id.h"
 #include "submap.h"
 #include "veh_type.h"
+#include "game.h"
```

Comment:
The include of "cata_utility.h", I buy. But why all the other additional includes?

Reply:
I think I got something wrong with my merge... I only intentionally added cata_utility.h